tap_eloqua/sync.py

```diff
@@ -377,7 +377,7 @@ def sync_activity_stream(client,
                          activity_type):
     finished = False
     sync_start = pendulum.now('UTC')
-    end_date = sync_start
+    end_date,last_date = sync_start,None
```

Comment:
Please rename `end_date, last_date` variables appropriately since they mean the same.  Also take care of formatting like below, 
```suggestion
    end_date, last_date = sync_start, None
```

Reply:
this looks like the last sync date, so changing it to `last_sync_date` but this part of code is not impacted by dev mode changes